libmamba/tests/src/core/test_configuration.cpp

```diff
@@ -538,6 +538,42 @@ namespace mamba
                               - https://repo.mamba.pm/conda-forge)"));
             }
 
+            TEST_CASE_METHOD(Configuration, "envs_dirs")
+            {
+                // Load default config
+                config.load();
+
+                // `envs_dirs` should be set to `root_prefix / envs`
+                const auto& envs_dirs = config.at("envs_dirs").value<std::vector<fs::u8path>>();
+                REQUIRE(envs_dirs.size() == 1);
+                REQUIRE(
+                    envs_dirs[0]
+                    == util::path_concat(
+                        std::string(config.at("root_prefix").value<mamba::fs::u8path>()),
+                        "envs"
+                    )
+                );
+            }
+
+            TEST_CASE_METHOD(Configuration, "envs_dirs_with_additional_rc")
+            {
+                std::string cache1 = util::path_concat(util::user_home_dir(), "foo_envs_dirs");
+                std::string rc1 = "envs_dirs:\n  - " + cache1;
+
+                load_test_config(rc1);
+
+                // `envs_dirs` should be set to the configured value `cache1`
+                // and `root_prefix / envs`
+                REQUIRE(
+                    config.dump()
+                    == "envs_dirs:\n  - " + cache1 + "\n  - "
+                           + util::path_concat(
+                               std::string(config.at("root_prefix").value<mamba::fs::u8path>()),
+                               "envs"
+                           )
```

Comment:
Nitpick: can this value be referenced by a variable in those two tests to ease the reading?